org.eclipse.ui.console/plugin.xml

```diff
@@ -82,13 +82,13 @@ M4 = Platform-specific fourth key
            commandId="AnsiConsole.command.copy_with_escapes"
            contextId="org.eclipse.ui.console.ConsoleView"
            schemeId="org.eclipse.ui.defaultAcceleratorConfiguration"
-           sequence="M1+Shift+C">
+           sequence="M1+M2+C">
      </key>
      <key
            commandId="AnsiConsole.command.copy_with_escapes"
            contextId="org.eclipse.ui.console.ConsoleView"
            schemeId="org.eclipse.ui.defaultAcceleratorConfiguration"
-           sequence="M1+Shift+Insert">
+           sequence="M1+M2+Insert">
```

Comment:
Please undo, Ctrl+Alt is still not OK, same above

Reply:
M2 is Shift, not Alt, see line 38.
And the option is still commented.
So it exactly like before, but (if ever enabled) will use the recommended M2 instead of Shift.